<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="fightpage">
    <widget class="QLabel" name="hero">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>500</y>
       <width>135</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images/diz.qrc">:/hero.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="hero_shield">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>610</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>护盾</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="hero_blood">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>580</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="hero_diz">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>540</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>战士</string>
     </property>
    </widget>
    <widget class="QLabel" name="monster">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>135</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images/diz.qrc">:/slime.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="mon_name">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>史莱姆</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="mon_blood">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>80</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="mon_shield">
     <property name="geometry">
      <rect>
       <x>781</x>
       <y>124</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>护盾</string>
     </property>
    </widget>
    <widget class="QLabel" name="skill1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>265</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images/diz.qrc">:/skill4.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="skill3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>170</y>
       <width>268</width>
       <height>207</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images/diz.qrc">:/skill5.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="skill2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>264</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images/diz.qrc">:/skill1.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="skill4">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>380</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images/diz.qrc">:/huixuanbiao.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>570</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>结束回合</string>
     </property>
    </widget>
    <widget class="QWidget" name="winpage" native="true">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>500</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>LV Up!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>You Win!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>血量上限+5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>进入下一关</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="msk">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="mnsk">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="skill5">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>210</y>
       <width>270</width>
       <height>310</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../images/diz.qrc">:/touzhi.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bagpage">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>10</y>
       <width>451</width>
       <height>661</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>651</width>
       <height>651</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images/diz.qrc"/>
 </resources>
 <connections/>
</ui>
